app/src/test/java/com/kickstarter/viewmodels/CrowdfundCheckoutViewModelTest.kt

```diff
@@ -359,8 +361,10 @@ class CrowdfundCheckoutViewModelTest : KSRobolectricTestCase() {
         viewModel.userChangedPaymentMethodSelected(cards.first())
         backgroundScope.launch(dispatcher) {
             viewModel.provideScopeAndDispatcher(this, dispatcher)
-            viewModel.pledgeOrUpdatePledge()
-
+            viewModel.pledgeOrUpdatePledge(
```

Comment:
@jlplks question here, this test has nothing to do with plot (or seems not configured for it), why is here `isIncremental=true`